libs/utils/trace_analysis.py

```diff
@@ -934,6 +930,9 @@ def getClusterFrequencyResidency(self, cluster):
         total_time = total_time.groupby(['frequency']).sum()
 
         ### Compute ACTIVE Time ###
+        # Assumption: all CPUs in a cluster run at the same frequency, i.e. the
```

Comment:
Maybe add that this assumption is verified by the Trace analysis module, when the trace is parsed.